packages/medusa/src/api/routes/admin/users/update-user.ts

```diff
@@ -18,6 +18,8 @@ import { EntityManager } from "typeorm"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminUpdateUserRequest"
+ * x-codegen:
+ *   method: update
```

Comment:
Client creates Request alias to type `role`. Not a problem for OAS codegen.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/users.ts#L79-L83